docs/content/about-iota/iota-wallet/getting-started.mdx

```diff
@@ -85,3 +85,106 @@ To use your IOTA Wallet on different devices or browsers, you can import it usin
+
+### Find Balances
+
+Once you have [accessed the Balance Finder](#how-to-access-the-balance-finder), hit the `Search` button and the
+Balance Finder will retrieve any sub-accounts related to your main account. 
+
+   ![Balance Finder Search](/img/about-iota/iota-wallet/getting-started/balance-finder-hit-search.png)
+
+:::tip 
+
+If you still have any missing balances, hit `Search` again.
+
+:::
+   
+### Migrate Your Stardust Assets
+
+If you have NFTs or native tokens that need migration, you will see a button indicating that you need to migrate
+them.
+
+   ![Migration Button](/img/about-iota/iota-wallet/getting-started/migration-button.png)
+
+When you click on it, it prompt you to use the [IOTA Wallet
+Dashboard](../iota-wallet-dashboard/getting-started.mdx) to migrate them.
+
+   ![Migration Pop In](/img/about-iota/iota-wallet/getting-started/migration-pop-in.png)
+
+
+## Vesting & Staked Vesting
+
+If your account has vested tokens, you will see a button indicating that you can access Vested Staking
+
+   ![Migration Button](/img/about-iota/iota-wallet/getting-started/vested-staking-button.png)
+
+When you click on it, it prompt you to use the [IOTA Wallet
+Dashboard](../iota-wallet-dashboard/getting-started.mdx) to manage your vested tokens.
+
+   ![Migration Pop In](/img/about-iota/iota-wallet/getting-started/vested-staking-pop-in.png)
+
+## What's Changed?
+
+### Account Management
+
+In Firefly, accounts were structured using a multi-account index, 
+which allowed generating multiple addresses under a single account without exposing the underlying BIP32 derivation
+paths. In practice, this means that Firefly would display a single wallet account that actually represented `n` addresses.
+
+The IOTA Wallet now each address individually.
+This change provides transparency for users managing accounts across different wallets or tools.
+
+When restoring accounts by importing a mnemonic phrase or seed,
+the IOTA Wallet will display the main account by default.
+To access derived accounts use the [Balance Finder](#use-the-balance-finder) tool to scan for assets associated with historical paths.
+This ensures compatibility with Firefly’s legacy multi-account framework. 
+
+:::note Legacy Chrysalis Accounts
+
+If you are trying to restore an Legacy Chrysalis-era account from a mnemonic in a third-party wallet, you may notice a
+discrepancy in the amount of IOTA in your wallet. You should use the [Balance Finder](#use-the-balance-finder) in the
+IOTA Web Wallet and move all your funds in `Legacy` addresses to a new address to be able to access them in any wallet.
+
+:::
+
+### Stronghold Has Been Deprecated
+
+:::danger 
+
+Restoring a wallet account using a seed is not an industry standard, it may not work on wallets that are not maintained
+by the IOTA Foundation. Once you restore your account using the seed, make sure to [export your mnemonic](./how-to/multi-account.mdx#export-a-mnemonic-for-an-account).
```

Comment:
```suggestion
Restoring a wallet account using a seed is not an industry standard, it may not work on wallets that are not maintained by the IOTA Foundation. Once you restore your account using the seed, you should transfer your funds to an account with a known mnemonic
```